bower.json

```diff
@@ -22,7 +21,7 @@
   ],
   "license": "Apache License, v2.0",
   "dependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
Okay, @liquidpele, looks like D3 added the ability to specify either an accessor or a comparator to `d3.bisector()`.  Our tests, and code, always pass an additional parameter, `i`, which under v3.5 causes it to be considered a comparator.